src/crypto/index.js

```diff
@@ -371,7 +371,9 @@ Crypto.prototype.isKeyBackupTrusted = async function(backupInfo) {
             try {
                 await olmlib.verifySignature(
                     this._olmDevice,
-                    backupInfo.auth_data,
+                    // verifySignature modifies the object so we need to copy
```

Comment:
This is fine as a workaround, but maybe `verifySignature` should also be changed to not do that...? (It seems like other might also be confused by it...)

Reply:
Yeah - I'm slightly hesitant to do it without fully understanding why it was done like that, although I suspect just for speed.